### DISCUSSION TITLE
How to force string with content 'n' to be stringify with quote

### DISCUSSION THREAD

**Discussion by extrame** [eemeli/yaml#375] [Q&A] [answered]
My json object has a string named "name" and with content 'n'.

It is stringified as `name: n`

which is pared back by yaml golang library as bool which I confirmed by specification is right.

So How can I stringify it as 

`name: "n" `

or

`name: 'n'`

to avoid this probolem?

**Comment by eemeli** [comment, marked as answer]
Probably best to use the `version: '1.1'` option, as that'll not only account for this, but other schema incompatibilities as well.
